packages/forma-36-website/src/components/Header.js

```diff
@@ -6,7 +6,7 @@ import { Link } from 'gatsby';
 const styles = {
   header: css`
     display: flex;
-    background-color: ${tokens.colorBlueDark};
+    background-color: ${tokens.colorBlueMid};
```

Comment:
Color adjusted with the top side of the page on the main view? It looks strange to me, that those 2 elements have different shades of blue.